src/odemis/driver/actuator.py

```diff
@@ -1882,17 +1882,6 @@ def _doMoveAbs(self, pos):
         position = position + self._metadata[model.MD_POS_COR]
         logging.debug("Moving axis %s (-> %s) to %g", self._axis, self._caxis, position)
 
-        # do referencing after i=5 moves or when actuator has accumulated 2pi steps
```

Comment:
Please leave this comment about why doing it every 5 moves.

Reply:
Well, I meant, put it _back_.

Reply:
ups :D